src/interfaces/IMetaMorpho.sol

```diff
@@ -95,8 +95,7 @@ interface IMetaMorphoBase {
     /// @notice Revokes the pending cap of the market defined by `id`.
     function revokePendingCap(Id id) external;
 
-    /// @notice Submits a forced market removal from the vault, potentially losing all funds supplied to the market.
-    /// @dev Warning: Submitting a forced removal will overwrite the timestamp at which the market will be removable.
+    /// @notice Submits a forced market removal from the vault, eventually losing all funds supplied to the market.
```

Comment:
I would not change `potentially` into `eventually`, the funds will not necessarily be lost if the timelock is reached and the market effectively removed: funds could possibly be recovered later with a reallocate if the market allows it

Reply:
This will be addressed in #354 pls don't change this PR let's merge it